plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -85,9 +88,15 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
   /** The layer created by this manger. Annotations will be added to this layer.*/
   internal var layer: L? = null
 
-  /** The source created by this manger. Feature data will bed added to this source.*/
+  /** The source created by this manger. Feature data will be added to this source.*/
```

Comment:
```suggestion
  /** The source created by this manager. Feature data will be added to this source.*/
```
here and in other places